src/index.ts

```diff
@@ -1,106 +1,142 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-import axios, { AxiosPromise, AxiosRequestConfig, Method } from 'axios';
-import axiosRetry from 'axios-retry';
-import * as fastq from 'fastq';
+import fastq from "fastq";
+import fetchRetry from "fetch-retry";
+import packageJson from "../package.json" assert { type: "json" };
 
-import { VERSION } from './version';
+const API_URL = "https://api.zenrows.com/v1/";
 
-const API_URL = 'https://api.zenrows.com/v1/';
+type HttpMethods = "GET" | "OPTIONS" | "POST" | "PUT" | "PATCH" | "DELETE";
```

Comment:
The API only supports `"GET" | "POST" | "PUT"`

Reply:
Done

Reply:
Are we interested in supporting it ? I can add it

Reply:
Less usage than POST or GET, so not a prio. You can leave as-is for now and create another Issue to add it.